.github/workflows/semantic-pull-request.yml

```diff
@@ -0,0 +1,55 @@
+name: Validate PR title
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+defaults:
+  run:
+    shell: bash -x -e -u -o pipefail {0}
+
+permissions:
+  pull-requests: read
+
+jobs:
+  semantic-pull-request:
+    # This job validates that the title of this pull-request matches the 
+    # [conventional-commit-types](https://github.com/commitizen/conventional-commit-types/blob/master/index.json).
+    # We use squash-based merging, so the PRs semantic title will become
+    # the squashed commit message's title.
+    # This allows follow-up actions, like automated changelog-generation.
+    # Also, standardized and beautiful commit messages!
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Semantic PR title
+        uses: amannn/action-semantic-pull-request@v5
```

Comment:
Should we configure this as a form of documentation of what we expect?

https://github.com/amannn/action-semantic-pull-request?tab=readme-ov-file#configuration

Reply:
Yes I was also split between implicitly loading the standard that’s defined up-stream and re-defining it for a more explicit definition. 

Let’s make it easier for our users by re-defining it